projects/plugins/jetpack/modules/copy-post.php

```diff
@@ -39,6 +42,26 @@ public function __construct() {
 		}
 	}
 
+	/**
+	 * Echos inline styles for the Jetpack logo.
+	 *
+	 * @return void
+	 */
+	public function print_inline_styles() {
+		echo '
```

Comment:
Should we add a condition based off `get_current_screen()` here, so this only gets added on the edit screen?

Reply:
I'm in favor of the underlying motivation for the idea (i.e., "let's avoid printing these styles unless appropriate"), but would such a check be redundant to the `( 'edit.php' === $GLOBALS['pagenow'] )` conditional on line 32?

Reply:
Ah, I missed that one, my bad! All good then! 🥳